### PR TITLE
distributed DQN using colossalai

### PR DESCRIPTION
**Pull request by number1roy**
Add  a colossalai naive DQN agent example that can use data-parallel for training

### CODE REVIEW

File: reinforcement learning/DQN/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+torch
+torchvision
+colossalai
```
Comment (feifeibear) [inline reinforcement learning/DQN/requirements.txt:3]: assign a version to colossalai

Comment (number1roy) [reply reinforcement learning/DQN/requirements.txt]: done

File: reinforcement learning/DQN/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+network structure which forwards with their own transitions data but backwards with global all-reduced gradient.
+Therefore, keeping batch size same but using multiple GPUs for data-parallel to train this example is similar to
+enlarge batch size. If you want to use the same batch size, please reduce your batch size to `batch_size / num_gpus`
+in your configs. However, if you are going to use larger batch size, please scale your learning rate and reduce
+the `total_step` in your configs for better performance.
+
+
+## Other features of colossalai
+As I mentioned before, using tensor-parallel, pipeline or ZeRO in training DQN is less effective. However, if you want to
+explore other features usage, you could check https://github.com/hpcaitech/ColossalAI-Examples/blob/main/language/gpt/train_gpt.py#L52
```
Comment (binmakeswell) [inline reinforcement learning/DQN/README.md:44]: The link can be given in a more elegant way : ) eg. ([text](link))

https://www.colossalai.org/docs/features/1D_tensor_parallel.
 https://github.com/hpcaitech/ColossalAI-Examples/blob/main/language/gpt/train_gpt.py#L52

**Comment by binmakeswell** [comment]
Hi  @number1roy Could you cancel the review request for feifeibear? This prevents other reviewers' action.

**Pull request closed without merging**